docs/api-docs/headless/tutorial.mdx

```diff
@@ -0,0 +1,722 @@
+# Headless Tutorial
+This tutorial will walk you through [creating a cart](#creating-a-cart), [checkout](http://localhost:3000/preview#creating-a-checkout), an [order](http://localhost:3000/preview#creating-an-order), and processing a payment using the [Carts](/docs/rest-management/carts/carts-single#create-a-cart) and [Checkout](https://developer.bigcommerce.com/docs/rest-management/checkouts) Rest Management APIs to build a headless storefront. 
+
+<Callout type="info">
+You can use REST Storefront APIs if you are using a Stencil theme; see the [Working with the REST Storefront Cart an Checkout APIs](/api-docs/storefront/tutorials/working-sf-apis) tutorial for more information, or use the [Storefront GraphQL APIs](/api-docs/storefront/graphql/carts-and-checkout) to build your headless site. 
+</Callout>
+
+This article assumes you have familiarity with the following concepts:
+
+* Creating a channel and channel site.
+* Retrieving product data using the GraphQL Storefront API calls 
```

Comment:
Are we wanting this to be about GraphQL usage or REST? I ask because this tutorial sits inside of the graphql folder, starts like you are going to learn about REST API usage, then mentions GraphQL here, before switching back to REST.

By default, we lean towards using GraphQL for every storefront type. REST makes more sense when you want to override pricing on carts or add metadata that GraphQL doesn't support yet (it's more server-to-server focused that GraphQL). So seems like we should make all the requests here GraphQL, then create another tutorial about using our REST APIs with these requests in a server-to-server context.

Reply:
Yep it's find to plan that for a future sprint. The change I'd make now is replacing "GraphQL" with "REST" within "Retrieving product data using the GraphQL Storefront API calls", as this article is using the REST APIs.

Reply:
Thanks @becomevocal. I made the change.